src/sonic-yang-models/yang-models/sonic-bgp-device-global.yang

```diff
@@ -59,6 +59,25 @@ module sonic-bgp-device-global {
                     default "unisolated";
                 }
 
+                leaf received_bandwidth {
+                    description "Configures device received-bandwidth state";
```

Comment:
Received-bandwidth is not a state. Could you may be rephrase as "Configures how device handles the received link-bandwidth extended community"?

Reply:
Sounds good!